### DISCUSSION TITLE
Recommend an English-language resource

### DISCUSSION THREAD

**Discussion by stepan323446** [lnreader/lnreader#821] [General]
All I've seen are non-working resources. 

 Some did not work, others had terrible and unsorted names (sss.., s.s15....).  

Still others didn’t have my light novels - eighty-six, the rising of the shield hero etc... Please tell me which ones to choose.

**Comment by rajarsheechatterjee** [comment]
The app doesn't recommend any sources. You can use the web to search for sites which have the novel. If the site is not present in the app you can open a source request.
